.editorconfig

```diff
@@ -9,3 +9,4 @@ insert_final_newline = true
 end_of_line = lf
 # editorconfig-tools is unable to ignore longs strings or urls
 max_line_length = null
+format_on_save = true
```

Comment:
This rule `insert_final_newline = true` should be added in order to auto-insert new line on save. 

Please see this answer for why it is important to always add a new line at the end of a file https://unix.stackexchange.com/a/18789

Reply:
Cool. I'll add this in the config.

Reply:
Doesn't this affect the python guys as well. This is a team wide decision. I'll need more clarity on why we need this

Reply:
@nwochaadim I've added a link to my comment on why it's important. Please find it for your information

Reply:
Noted @adesege that's a team-wide decision. You can bring this up whenever we have a team-wide meeting. For now, you can remove this and the `format_on_save` option too @brayoh